app/helpers/cornell_catalog_helper.rb

```diff
@@ -1432,31 +1432,27 @@ def sub_tilde_path(var,id,aeon_codes,document)
   end
 
 # (group == "Circulating" ) ? blacklight_cornell_request.magic_request_path("#{id}") :  "http://wwwdev.library.cornell.edu/aeon/monograph.php?bibid=#{id}&libid=#{aeon_codes.join('|')}"
-  def request_path(group,id,aeon_codes,document,scan)
-	id_scan = scan == "yes" ? "#{id}.scan" : "#{id}"
+  def request_path(group, id, aeon_codes, document, scan)
+    id_scan = scan ? "#{id}.scan" : "#{id}"
```

Comment:
I'm guessing this means that `scan` won't ever have a value that should be treated as no/false?

Reply:
Oops, I see your code below now. Ignore this. Great refactor.